lib/utils.pm

```diff
@@ -711,6 +712,19 @@ sub handle_login {
     send_key "ret";
 }
 
+sub service_action {
```

Comment:
As we are having more and more util functions. Could you add some documentation comments? E.g. take testapi.pm as reference